lib/docker-toolbelt.d.ts

```diff
@@ -0,0 +1,26 @@
+import { static } from 'express';
```

Comment:
I think you imported this by accident?

Reply:
Yup, it seems vscode thought it was being helpful :confused: